bpytop.py

```diff
@@ -1846,7 +1855,7 @@ def _draw_fg(cls):
 			stats = net.stats[net.nic][direction]
 			if stats["redraw"] or cls.resized:
 				if cls.redraw: stats["redraw"] = True
-				Graphs.net[direction] = Graph(w - bw - 3, cls.graph_height[direction], THEME.gradient[direction], stats["speed"], max_value=stats["graph_top"], invert=False if direction == "download" else True)
+				Graphs.net[direction] = Graph(w - bw - 3, cls.graph_height[direction], THEME.gradient[direction], stats["speed"], max_value=stats["graph_top"], invert=False if direction == "download" else True, color_max_value=stats["top"])
```

Comment:
Using stats["top"] value for the color is gonna look weird until you have actually reached a high value since stats["top"] only records the highest reached value for current session.